### PR TITLE
[TS-166] example code for packet of web

### PR DESCRIPTION
**Pull request by jenchik**

### CODE REVIEW

File: examples/web/main.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		},
+		Debug: true,
+	})
+
+	e.GET("/test/:command", func(c web.Context) error {
+		cmd := c.Param("command")
+		fmt.Println("Received command:", cmd)
+
+		switch cmd {
+		case "nop":
```
Comment (im-kulikov) [inline examples/web/main.go]: O_o

**Review by im-kulikov — DISMISSED** [review]
pls, add comment line for `case "nop"`

File: examples/web/main.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	go func() {
+		if err := web.StartServer(e, os.Getenv("BIND")); err != nil {
+			if !strings.Contains(err.Error(), http.ErrServerClosed.Error()) {
+				e.Logger.Error(err)
+			}
+			ch <- syscall.SIGABRT
+		}
+	}()
+
+	// Wait for signals:
```
Comment (kpeo) [inline examples/web/main.go]: colon useless in comment here

**Comment by jenchik** [comment]
fixed

**Review by im-kulikov — DISMISSED** [review]
`e.GET("/test/:command", func(c web.Context) error {`
`web.Context` - is sounds strange

**Review by pcherednichenko — DISMISSED** [review]

**Review by pcherednichenko — APPROVED** [review]

**Review by kpeo — APPROVED** [review]

**Pull request merged**